### PR TITLE
Update instance manager

### PR DESCRIPTION
**Pull request by keithalucas**
Use an instance manager version with the tgt 100% CPU usage fix.

https://github.com/longhorn/longhorn/issues/3636

Signed-off-by: Keith Lucas <keith.lucas@suse.com>

### CODE REVIEW

**Review by innobead — CHANGES_REQUESTED** [review]
Also need to update chart files: questions.yml and values.yml

ref: https://github.com/longhorn/longhorn/commit/3301f9b9155ae4ee706efdcf625cbe4f01cb5554

P.S. do the same update for other backport PRs.

**Comment by keithalucas** [comment]
> Also need to update chart files: questions.yml and values.yml
> 
> ref: [3301f9b](https://github.com/longhorn/longhorn/commit/3301f9b9155ae4ee706efdcf625cbe4f01cb5554)
> 
> P.S. do the same update for other backport PRs.

I updated the helm chart here and in the other PRs.

File: chart/questions.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ questions:
     label: Longhorn Share Manager Image Repository
     group: "Longhorn Images Settings"
   - variable: image.longhorn.shareManager.tag
-    default: v1_20211020
+    default: v1_20220303
```
Comment (innobead) [inline chart/questions.yaml:68]: Should instance manager not share manager

**Review by innobead — CHANGES_REQUESTED** [review]

File: chart/questions.yaml (hunk 1 above)
Comment (keithalucas) [reply chart/questions.yaml]: Fixed

**Review by innobead — APPROVED** [review]

**Pull request merged**